internal/symbolz/symbolz.go

```diff
@@ -39,9 +39,9 @@ var (
 // separated by '+') and returns the symbolz output in a string. It
 // symbolizes all locations based on their addresses, regardless of
 // mapping.
-func Symbolize(sources plugin.MappingSources, syms func(string, string) ([]byte, error), p *profile.Profile, ui plugin.UI) error {
+func Symbolize(p *profile.Profile, force bool, sources plugin.MappingSources, syms func(string, string) ([]byte, error), ui plugin.UI) error {
 	for _, m := range p.Mapping {
-		if m.HasFunctions {
+		if m.HasFunctions && !force {
```

Comment:
newMapping() in symbolizer.go does this check as ```if !force && (m.HasFunctions || m.HasFilenames || m.HasLineNumbers) {```, should these be consistent?

Reply:
No, because symbolz can only provide functions. If we have functions but not filenames there is no point on redoing symbolz, while it might be worthwhile doing local symbolization.

Reply:
Ah, got it. It's somewhat subtle, consider adding a comment.